packages/components/package.json

```diff
@@ -45,6 +45,7 @@
     "shortid": "^2.2.16"
   },
   "peerDependencies": {
+    "@adobe/react-spectrum": "^3.x",
```

Comment:
I think this needs to be flagged as a breaking change? Any consumer of `@deephaven/components` will need to include `@adobe/react-spectrum` now, right? Or is it just if you try and use `SpectrumProvider`

Reply:
As far as I can tell, this doesn't cause any issues until attempting to consume something with a Spectrum dependency (currently only SpectrumProvider).

Reply:
@mofojed I'm not 100% sure whether this is a breaking change or not.
- I uninstalled @adobe/react-spectrum from enterprise and things still work (presumably due to local install in my community location
- I uninstalled from commmunity locally, but then the npm run start-community fails with compilation errors

It's quite possible in the real case where community has been deployed, enterprise will be fine if there are no references to SpectrumProvider, but I can't find a good way to actually test this theory.

I could see marking this as a breaking change to be safe unless you have any other ideas of how we can test it.

Reply:
I did some more testing, and it seems this is a breaking change regardless if SpectrumProvider is used or not. I've updated the description footer accordingly.